app/modules/ads/setup.js

```diff
@@ -241,6 +241,13 @@ export const adsSetup = {
       if (!isGeoEnabled('wgAdDriverPubMaticOutstreamCountries')) {
         context.remove('bidders.prebid.pubmatic.slots.incontent_player');
       }
+
```

Comment:
Same remarks as in https://github.com/Wikia/app/pull/16859/files

Reply:
Done!